main.cpp

```diff
@@ -108,6 +103,19 @@ Thing 4) Audio mixer channel strip
     3) control audio dynamics through the compressor
  */
 
+/*
+Thing 10) Airport
+5 properties:
+    1) runway
+    2) passengers terminal
+    3) control tower
+    4) fire fighters station
+    5) cargo apron
+3 things it can do:
```

Comment:
You're going to have a hard time coming up with 5 properties and 3 actions for each of these in part 1d.
- [ ] come up with a different `#10` that splits into sub-parts better.